dotcom-rendering/src/web/components/DynamicPackage.tsx

```diff
@@ -1,24 +1,68 @@
-import type { DCRContainerPalette } from '../../types/front';
+import type { DCRContainerPalette, DCRGroupedTrails } from '../../types/front';
 import { LI } from './Card/components/LI';
 import { UL } from './Card/components/UL';
 import { FrontCard } from './FrontCard';
 
 type Props = {
-	trails: TrailType[];
+	groupedTrails: DCRGroupedTrails;
 	containerPalette?: DCRContainerPalette;
 	showAge?: boolean;
 };
 
+const SnapRow = ({
+	snap,
+	containerPalette,
+	showAge,
+}: {
+	snap: TrailType;
+	containerPalette?: DCRContainerPalette;
+	showAge?: boolean;
+}) => {
+	return (
+		<UL direction="row">
+			<LI padSides={true} padBottom={true}>
+				<FrontCard
+					trail={snap}
+					starRating={snap.starRating}
+					containerPalette={containerPalette}
+					containerType="dynamic/package"
+					showAge={showAge}
+					headlineSize="large"
+					headlineSizeOnMobile="medium"
+					imagePosition="right"
+					imagePositionOnMobile="left"
+					imageSize="medium"
+					trailText={snap.trailText}
+				/>
+			</LI>
+		</UL>
+	);
+};
+
 export const DynamicPackage = ({
-	trails,
+	groupedTrails,
 	containerPalette,
 	showAge,
 }: Props) => {
-	const [primary, ...remaining] = trails;
+	// Take the first 'snap' - all others are treated as standards
+	const snap = groupedTrails.snap.shift();
```

Comment:
This may be fine but my gut tells me to avoid shift as it mutates the original array. Can this be rewritten somehow using `slice(0, 1)`?

Reply:
Had a fiddle around with this - I believe our tsconfig is set up so that typescript assumes any array access (e.g myArray[150]) will always assume that it is not undefined - not great for us. 

Because of this I moved to using .slice(0, 1) and keeping `snaps` as an array (even though we know it will only be 0 or 1 items). Not my favourite solution but not a lot of great options

Reply:
Update: Used array deconstructing with .shift to work around this instead